### PR TITLE
Tsw 53 pr this

### PR DESCRIPTION
**Pull request by CarbonBloem**

### CODE REVIEW

**Review by JPBeukes — CHANGES_REQUESTED** [review]
- Sending from EOS to Telos gives a transaction id link to teloscan, not bloks.io. I'm referring to this popup:
image
- Auto login (using local storage) has been disabled.
- Metamask does not prompt you to select a different network when you switch evm networks in the app.
- Sending tokens from Telos native to any of the EVM chains does not work. Might be a reporter issue, but sending from one of the EVM chains to Telos native works, so it might not.

File: src/pages/Bridge.vue [hunk 1]
```diff
@@ -28,20 +34,21 @@
...
 import metamask from "src/components/Metamask";
 import bridgeStepper from "../components/bridge/bridgeStepper.vue";
+import bridgeStepper2 from "../components/bridge/bridgeStepper2.vue";
 import bridgeDashboard from "src/components/bridge/TeleportDash.vue";
-import bridgeExternal from "src/components/bridge/ExternalBridge.vue";
 
 export default {
   components: {
     bridgeStepper,
+    bridgeStepper2,
```
Comment (Dewald928) [inline src/pages/Bridge.vue:43]: I assume bridge 1 is evm and stepper 2 is antelope? bit unclear

File: src/store/account/actions.js [hunk 2]
```diff
@@ -4,29 +4,34 @@ export const login = async function (
   { commit, dispatch },
   { idx, account, returnUrl }
 ) {
-  const authenticator = this.$ual.authenticators[idx];
+  const authenticator = this.$ual().authenticators[idx];
   try {
     commit("setLoadingWallet", authenticator.getStyle().text);
     await authenticator.init();
+    console.log(account);
```
Comment (Dewald928) [inline src/store/account/actions.js:11]: logs

**Review by Dewald928 — CHANGES_REQUESTED** [review]
Just a few questions and tweaks

File: src/components/bridge/ConnectTo.vue [hunk 3]
```diff
@@ -0,0 +1,289 @@
+<template>
```
[the remark this answers is not in the document]
Comment (CarbonBloem) [reply src/components/bridge/ConnectTo.vue:1]: It is used for the antelope section (bridgeStepper2)

File: src/components/bridge/Connect2.vue [hunk 4]
```diff
@@ -0,0 +1,298 @@
+<template>
```
[the remark this answers is not in the document]
Comment (CarbonBloem) [reply src/components/bridge/Connect2.vue:1]: It is used for the antelope section (bridgeStepper2)

File: src/pages/Bridge.vue (hunk 1 above)
Comment (CarbonBloem) [reply src/pages/Bridge.vue:43]: Yes, that is true. It is used for the antelope bridge.

File: src/store/account/actions.js (hunk 2 above)
Comment (CarbonBloem) [reply src/store/account/actions.js:11]: Will remove all the console logs in the project.

**Review by Dewald928 — APPROVED** [review]
All seems well.

**Pull request merged**